Formatted and Updated README.md - Added Streamlit based WebApp 👨‍💻✅

prateekralhan opened the pull request:
Hi @SamuelAlgheriniAI and @therealexpertai ,

Kudos to you for bringing up this speech-to-text tutorial using transformers and the NL API. I worked on developing a simple streamlit based webapp on the same and I think it will be fruitful to have it as a part of README here as the motivation behind developing this came from your work 😄! The entire webapp codebase has also been added as a separate folder with an elaborate README explaining how to setup it up.

Github Fork URL : https://github.com/prateekralhan/speech-2-data/tree/main/streamlit_webapp

You can find the entire webapp source-code in the **stream_app** directory of the repo.

Happy opensourcing!

Cheers,
Prateek

prateekralhan commented:
@SamuelAlgheriniAI / @therealexpertai , Do we have any comments or updates on this?

SamuelAlgheriniAI commented:
Hi Prateek!

Thank you very much, your implementation is great, I like it very much (and I’m not the only one!). I am very happy when other people improve the work done.

Unfortunately, the repo is corporate (it is not personal) and I can't add your commit in the main because of the use of streamlite.

Since, however, I believe (indeed we believe) that it is a valuable addition to the project, we think it would be useful to add the link to your project in the readme so that visitors can use your code to take full advantage of the program's potential.

What do you think? Does it work for you?

Samuel



***@***.***

Samuel Algherini | Technical Trainer
Via Ostiense 131/L Piano 5, Rome, RM 00154, Italy
Phone: +39 06 83361861
  www.expert.ai<http://www.expert.ai>
***@***.***<https://www.facebook.com/expertdotai/> ***@***.*** <https://www.linkedin.com/company/expert-ai/mycompany/>  ***@***.*** <https://twitter.com/Expertdotai>  ***@***.*** <https://www.youtube.com/user/expertsystemvideo>
***@***.***<https://www.expert.ai/resource/strong-performer-in-forrester-wave/?utm_medium=signature&utm_source=codetwo&utm_campaign=forrester-wave>
This corporate e-mail account is used only for work-related purposes; it is not guaranteed that any correspondence sent to this address will be read by the addressee only, as it may be necessary, under certain circumstances, for third parties appointed by the Undertaking to access this corporate e-mail account. Please do not send any messages of a personal nature to this address.

From: Prateek Ralhan ***@***.***>
Sent: mercoledì 30 novembre 2022 06:10
To: therealexpertai/speech-2-data ***@***.***>
Cc: Samuel Algherini ***@***.***>; Mention ***@***.***>
Subject: Re: [therealexpertai/speech-2-data] Formatted and Updated README.md - Added Streamlit based WebApp 👨‍💻✅ (PR #1)

Non si ricevono spesso messaggi di posta elettronica da ***@***.******@***.***>. Informazioni sul perché è importante<https://aka.ms/LearnAboutSenderIdentification>

@SamuelAlgheriniAI<https://github.com/SamuelAlgheriniAI> / @therealexpertai<https://github.com/therealexpertai> , Do we have any comments or updates on this?

—
Reply to this email directly, view it on GitHub<https://github.com/therealexpertai/speech-2-data/pull/1#issuecomment-1331650990>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ATGIM3TDXIPM54T36S62B7TWK3OQ3ANCNFSM6AAAAAASD3YAUM>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>


INTERNAL

prateekralhan commented:
@SamuelAlgheriniAI , thanks a lot for the great feedback. Happy to know that you liked your implementation.

I have updated the README accordingly and removed the webapp codebase directory as well.

You can validate the changes and accordingly merge the PR if it meets all the requirements.
